vertx-mongo-client/src/main/java/io/vertx/ext/mongo/impl/codec/json/JsonObjectCodec.java

```diff
@@ -106,6 +149,14 @@ protected Object readObjectId(BsonReader reader, DecoderContext ctx) {
 
   @Override
   protected Object readDateTime(BsonReader reader, DecoderContext ctx) {
```

Comment:
I'm for the update as it is now in the PR.
MongoDB internally doesn't save timezone information, so (at least currently) it doesn't matter if the value is sent as an ISO string or in miliseconds.